python.py

```diff
@@ -0,0 +1,20 @@
+# here is a coment
+
+#
+# block coment
+# thas a multiline
```

Comment:
Typo inside comment in word `thas`. Possible corrections:  `thus` (11), `than` (3), `has` (3)